doc/content/examples/example6a.md

```diff
@@ -0,0 +1,66 @@
+# Example 6a: Displacement controlled contact problem with elastic soil.
+
+## Model Description
+
+This example demonstrates frictional contact between a solid block and elastic soil in MASTODON. A 6 inch cubic block is centered on top of a 4 foot cubic block of soil. The top surface of the soil is free and the remaining surfaces are fixed in all directions. A uniform normal pressure of 5 psi is applied to the top of the smaller solid block and it is given a prescribed displacement in a direction parallel to the contact surface. The resulting normal and frictional forces at the interface of the two materials are then obtained and compared with analytical results.
+
+
+## Modeling in MASTODON
+
+!listing examples/ex06a/psf_grav.i
+
+The mesh [ex06input] used in this model was generated externally using the mesh generation software, [Cubit](https://cubit.sandia.gov/). MASTODON imports the mesh shown in [fig:ex06_input] using the `FileMesh` type specified in the `Mesh` block. Details regarding supported mesh file types can be found at [GettingStarted](manuals/user/index.md).
+
+!media media/examples/ex06/ex06_input.png
+       style=width:50%;margin-left:150px;float:center;
+       id=fig:ex06_input
+       caption=Input model in MASTODON.
+
+The displacement variables are defined in the `Variables` block. The accelerations, velocities, normal and tangential (frictional) forces are defined as auxiliary variables in the `AuxVariables` block.
+
+[TensorMechanics](syntax/index.md) and [InertialForce](syntax/index.md) kernels are used to model the dynamics of the 3D solid elements. [Gravity](syntax/index.md) kernels is used to apply the gravitational force. The [Newmark time integration](manuals/theory/index.md) parameters used in this problem correspond to the Newmark's average acceleration method, i.e. `beta = 0.25` and `gamma = 0.5` and damping is ignored.
```

Comment:
The Gravity block is specified in 'Kernels' in order to apply gravitational force to the entire model.